src/dbus/client/thread_api_dbus.cpp

```diff
@@ -150,6 +150,7 @@ DBusHandlerResult ThreadApiDBus::DBusMessageFilter(DBusConnection *aConnection,
     {
         f(role);
     }
+    return DBUS_HANDLER_RESULT_HANDLED;
```

Comment:
There should be one return statement per free function or method at the end of the free function or method.

Reply:
Thanks for review.
Can we fix it by creating a return value variable?
`DBusHandlerResult ThreadApiDBus::DBusMessageFilter(DBusConnection *aConnection, DBusMessage *aMessage)
{
    DBusHandlerResult handled = DBUS_HANDLER_RESULT_NOT_YET_HANDLED;
    .....
    .....
    handled = DBUS_HANDLER_RESULT_HANDLED;

exit:
    return handled;
}`

Reply:
Yes, exactly. Thanks.

Reply:
updated